src/tmx/TmxApi/tmx/messages/IvpJ2735.c

```diff
@@ -83,22 +83,22 @@ const char *getMessageSubTypeFromMsgId(e_DSRCmsgID msgId)
 {
 	switch(msgId)
 	{
-//		case DSRCmsgID_alaCarteMessage: return "ACM";
-		case DSRCmsgID_basicSafetyMessage: return "BSM";
-		case DSRCmsgID_basicSafetyMessageVerbose: return "BSMV";
-		case DSRCmsgID_commonSafetyRequest: return "CSR";
-		case DSRCmsgID_emergencyVehicleAlert: return "EVA";
-		case DSRCmsgID_intersectionCollisionAlert: return "IC";
-		case DSRCmsgID_mapData: return "MAP";
-		case DSRCmsgID_nmeaCorrections: return "NMEA";
-		case DSRCmsgID_probeDataManagement: return "PDM";
-		case DSRCmsgID_probeVehicleData: return "PVD";
-		case DSRCmsgID_roadSideAlert: return "RSA";
-		case DSRCmsgID_rtcmCorrections: return "RTCM";
-		case DSRCmsgID_signalPhaseAndTimingMessage: return "SPAT";
-		case DSRCmsgID_signalRequestMessage: return "SRM";
-		case DSRCmsgID_signalStatusMessage: return "SSM";
-		case DSRCmsgID_travelerInformation: return "TIM";
+//		case msgID_alaCarteMessage: return "ACM";
+		case msgID_basicSafetyMessage: return "BSM";
+		case msgID_basicSafetyMessageVerbose: return "BSMV";
+		case msgID_commonSafetyRequest: return "CSR";
+		case msgID_emergencyVehicleAlert: return "EVA";
+		case msgID_intersectionCollisionAlert: return "IC"; // I think this can be changed to intersectionCollision
```

Comment:
Not sure what the difference between the intersection Collision and its Alert. Do we need a separate entry for each?  Is this a TODO? can we provide detail how to update this?